README.md

```diff
@@ -5,6 +5,8 @@ This repository serves as a check-in for attendees of the **Git Workshop for Mod
 ---
 ## Attendees
 
+Victor Edwin E Reyes

```

Comment:
Just curious, why not add the period after the middle initial?
```suggestion
* Victor Edwin E. Reyes
```

Reply:
It's an old tradition I picked up from my days in the corps. There, if I remember correctly, you only put periods after middle initials if the person was dead. In fact even abbreviations in ranks and titles lack periods (e.g. C2Lt Victor Edwin E Reyes 3Cl). 

Of course, I've resigned now, but I still just never put periods after any living person's middle initials out of habit. That and it's just a fun check if the person parsing is paying attention (as in this case!). Though, I'd gladly put one down in line with the requirements of any regex parser or other formatting requirement.